src/thing_stats.c

```diff
@@ -615,12 +645,36 @@ long calculate_correct_creature_pay(const struct Thing *thing)
     struct CreatureControl* cctrl = creature_control_get_from_thing(thing);
     struct CreatureStats* crstat = creature_stats_get_from_thing(thing);
     long pay = compute_creature_max_pay(crstat->pay, cctrl->explevel);
-    // If torturing creature of that model, halve the salary
-    if (dungeon->tortured_creatures[thing->model] > 0)
-        pay /= 2;
+    // If torturing creature of that model, halves the salary if the rule is enabled.
+    if ((dungeon->tortured_creatures[thing->model] > 0) && (game.conf.rules.game.torture_payday > 0))
+        pay *= (game.conf.rules.game.torture_payday / 100);
     return pay;
 }
 
+long calculate_correct_creature_training_cost(const struct Thing *thing)
+{
+    struct Dungeon* dungeon = get_dungeon(thing->owner);
+    struct CreatureControl* cctrl = creature_control_get_from_thing(thing);
+    struct CreatureStats* crstat = creature_stats_get_from_thing(thing);
+    long training_cost = compute_creature_max_training_cost(crstat->training_cost, cctrl->explevel);
+    // If torturing creature of that model, halves the training cost if the rule is enabled.
+    if ((dungeon->tortured_creatures[thing->model] > 0) && (game.conf.rules.game.torture_training_cost > 0))
```

Comment:
oh actually since it's now a percentage, settings it to 100 will "disable" it so I don't need to check this condition anymore and I can set torture_training_cost and torture_scavenging_cost to 100 by default, will do it later🤔